selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/NavigateToFileTest.java

```diff
@@ -147,6 +148,9 @@ private void launchNavigateToFileAndCheckResults(
     String openedFileNameInTheTab = openedFileWithExtension.replace(".java", "");
     launchNavigateToFileFromUIAndTypeValue(navigatingValue);
     navigateToFile.waitSuggestedPanelIsDisplayed();
```

Comment:
According to [scrrenshot](https://user-images.githubusercontent.com/7760565/34935870-055e2b46-f9e8-11e7-986b-e07d584c9f16.png) from the target issue the problem seems with method *waitSuggestedPanelIsDisplayed()* which has succeed before suggested panel is actually displayed. Let's fix the method instead of adding `WaitUtils.sleepQuietly(3)` pause.